mappings/net/minecraft/obfuscate/DontObfuscate.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/obfuscate/DontObfuscate net/minecraft/obfuscate/DontObfuscate
+	COMMENT This annotation applies to some unobfuscated elements within the Minecraft
+	COMMENT source code.
+	COMMENT
+	COMMENT <p>Its behavior appears as follows:
+	COMMENT <ul>
+	COMMENT <li>The annotation itself is not obfuscated.</li>
+	COMMENT <li>If a class is annotated, it is not obfuscated.</li>
+	COMMENT <li>If a member is annotated, it and its containing class is not obfuscated,
+	COMMENT but other members in the same class may stay obfuscated.</li>
+	COMMENT </ul>
+	COMMENT
+	COMMENT <p>In addition, single-abstract-method interfaces used as lambda expressions
+	COMMENT have their single abstract method unobfuscated without this annotation.
+	COMMENT
+	COMMENT <p>This annotation is not {@link java.lang.annotation.Documented}, and hence
+	COMMENT will not appear in the generated javadoc for annotated elements.
+	COMMENT
+	COMMENT @see net.minecraft.client.Main#main(String[])
+	COMMENT @see net.minecraft.data.Main#main(String[])
+	COMMENT @see net.minecraft.server.Main#main(String[])
+	COMMENT @see net.minecraft.client.ClientBrandRetriever#getClientModName()
+	COMMENT @see net.minecraft.server.MinecraftServer#getServerModName()
+	COMMENT @see net.minecraft.util.CubicSampler.RgbFetcher
+	COMMENT @see net.minecraft.world.level.ColorResolver#getColor()
```

Comment:
This is way too many usages in the docs, if I want to find all usages I have intellij. Probably better to just list one or two of them.

Reply:
Hmm, this becomes a list when rendered, so it shouldn't be too bad? Also when browsing pure javadoc on web, you can't have access to the code unfortunately. I should test how the javadoc use page works with this. If they are shown in the use page, I will definitely cut it. Otherwise, keeping this long list isn't really problematic.

Reply:
Have cut it already to only include the rgb fetcher and color resolver.